umpost/conversion_driver_esm1p5.py

```diff
@@ -259,15 +326,18 @@ def convert_esm1p5_output_dir(esm1p5_output_dir):
         warnings.warn(
             f"No files matching pattern '{fields_file_name_pattern}' "
             f"found in {current_atm_output_dir.resolve()}. No files will be "
-            "converted to NetCDF."
+            "converted to netCDF."
         )
 
         return [], []  # Don't try to run the conversion
 
-    succeeded, failed = convert_fields_file_list(
-        atm_dir_fields_files,
-        nc_write_dir
-    )
+    input_output_pairs = []
```

Comment:
@truth-quark I've added a loop here where `get_ff_date` is called, and the list of input/output paths is constructed. Let me know if this structure seems reasonable!